### ISSUE TITLE
Rewrite weather configuration

### ISSUE THREAD

**Issue by fluxer** [fluxer/katana#26]
As it is now the user has to brute-force guess the weather station name by entering possible matches instead of being presented with station choices. Choices can be presented based on country or distance (latitude and longitude via gelocation), assuming providers provide such information, if such information is not provided then provider choices should be reconsidered.

Guessing the station name based on IP geolocation is also far from perfect as it can narrow down very few use cases, e.g. geolocation data is for Ruse/Pleven but station is in Sofia.

Bottom line is configuration has to be scrapped and redone to be even close to usefull to the end-user, it affects two Plasma applets and one wallpaper plugin.

**Comment by fluxer** [comment]
Closing due to https://github.com/fluxer/kde-extraapps/commit/e4b67f8165d6d2ae562a736b314fdc4a0792842e.
